### DISCUSSION TITLE
Congrats (Re: JayzTwoCents video)

### DISCUSSION THREAD

**Discussion by MailYouLater** [Rem0o/FanControl.Releases#682] [General]
Hello again @Rem0o,
    I see JayzTwoCents featured your FanControl software in a video[(link)](https://youtu.be/uDPKVKBMQU8)  recently.  Congratulations on getting publicly recognized by someone who has a large enough audience that him 'spreading the word' could potentially cause a notable increase in people using your project, he seemed pretty impressed.  Anyway, we all really appreciate what you've done (and continue to do) with this project, so please keep up the good work.  😁

Thank you.

**Comment by MailYouLater** [comment]
(P.S.  I still hold out hope that one day, you might open source this great project.  Remember the contrast of what happened to SpeedFan, as opposed to OpenHardwareMonitor.  LibreHardwareMonitor extended the legacy of OHM and continues to provide improved functionality over it, but there was nothing the community could do for SpeedFan 😢.)

**Reply by Rem0o** [reply]
Thansk!

See https://github.com/Rem0o/FanControl.Releases/issues/89 for open-source discussion.

TLDR: The part that matters, aka the hardware compatibility, is already open-source.

**Reply by MailYouLater** [reply]
> The hardware compatibility, is already open-source.

I know, remember, I tracked you down and asked you about it in an LHM issue.  I understand your reasons, and I won't try to argue the point, but I like open source software, and I think you understand (at least a little bit) where I'm coming from with that, or you probably wouldn't have submitted code to help out the LHM project, and likely wouldn't have released FanControl as you have.  So even if you have no such plans now, I still have hope that someday you might change your mind.  I won't hold my breath, but I do still have hope.

Anyway, the point of this was to congratulate you, both on doing a good job with your continued efforts, and on getting some public recognition.

**Comment by Mandoscottie** [comment]
@Rem0o just registered to say thank you for such a superb utility, i flung over a few bucks via Paypal to say thanks and to support your endevour.

Was awesome to see Jay publicise your work.

keep up the superb work, I'm converting everyone i can :) so simple an idea, yet nothing else comes close to the granular control FC offers.
